### PR TITLE
Initial Dynamic Layouts

### PR DESCRIPTION
**Pull request by hey-waffles**
Creates the initial framework for dynamic layouts. This is by no means done, but by merging this in we can utilize parts of it (for example, molecules and atoms) in other portions of the site, such as the articles

### CODE REVIEW

File: src/components/design/forms/Multiselect.tsx [hunk 1]
```diff
@@ -0,0 +1,6 @@
+import React from "react";
+import { Select } from "./Forms";
+
+export function Multiselect(props: any): JSX.Element {
```
Comment (hey-waffles) [inline src/components/design/forms/Multiselect.tsx:4]: Comment this

File: src/components/layouts/Layouts.tsx [hunk 2]
```diff
@@ -0,0 +1,161 @@
+
+import React from "react";
+import { Formik, Form as FormikForm } from "formik";
+import { Card, Col, Container, Row, Tab, Nav, Form, Button } from "react-bootstrap";
+import { Entity } from "../../types/documents";
+import { DynamicMolecule } from "./molecules";
+import { Section } from "../../types/layouts/section";
+import { Atom, Molecule, Page } from "../../types/layouts";
+
+interface DynamicLayoutProps {
```
Comment (hey-waffles) [inline src/components/layouts/Layouts.tsx:10]: Comment this

File: src/components/layouts/Layouts.tsx [hunk 3]
```diff
@@ -0,0 +1,161 @@
...
+  entity: Entity;
+  onSubmit?: (values: Record<string, unknown>) => void;
+}
+
+function DynamicSubsection(props: any) {
```
Comment (hey-waffles) [inline src/components/layouts/Layouts.tsx:15]: Comment this

File: src/components/layouts/Layouts.tsx [hunk 4]
```diff
@@ -0,0 +1,161 @@
...
+          <Row>
+            {contents}
+          </Row>
+        </Card.Body>
+      </Card>
+    </Col>
+  );
+}
+
+export function DynamicSection(props: any) {
```
Comment (hey-waffles) [inline src/components/layouts/Layouts.tsx:38]: Hey, maybe comment this?

File: src/components/layouts/Layouts.tsx [hunk 5]
```diff
@@ -0,0 +1,161 @@
...
+      </Container>
+    </Tab.Pane>
+  );
+}
+
+/**
+ * 
+ * @param props 
+ */
+export function DynamicFormWrapper(props: any) {
```
Comment (hey-waffles) [inline src/components/layouts/Layouts.tsx:85]: Finish commenting this

File: src/components/layouts/Layouts.tsx [hunk 6]
```diff
@@ -0,0 +1,161 @@
...
+export function DynamicPages(props: any) {
+  const dynamicPages: JSX.Element[] = [];
+
+  props.dynamicLayout.pages.forEach((page: Page) => {
+    dynamicPages.push(
+      <DynamicPage page={page}/>
+    );
+  });
+
+
```
Comment (hey-waffles) [inline src/components/layouts/Layouts.tsx:117]: Be sure to run the linter too <3

File: src/components/layouts/atoms/AtomError.tsx [hunk 7]
```diff
@@ -0,0 +1,10 @@
+import React from "react";
+import { Col } from "react-bootstrap";
+
+export function AtomError(props: any) {
```
Comment (hey-waffles) [inline src/components/layouts/atoms/AtomError.tsx:4]: Comment

File: src/components/layouts/atoms/TernaryInput.tsx [hunk 8]
```diff
@@ -0,0 +1,9 @@
+import React from "react";
+import { Button } from "react-bootstrap";
+import { AtomProps } from "../../../types/layouts/atom";
+
+export function TernaryInput(props: AtomProps) {
```
Comment (hey-waffles) [inline src/components/layouts/atoms/TernaryInput.tsx:5]: Comment this sorry thing

File: src/components/layouts/molecules/Accordion.tsx [hunk 9]
```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import { Accordion, Button, Row } from "react-bootstrap";
+
+function AccordionToggle(props: any) {
```
Comment (hey-waffles) [inline src/components/layouts/molecules/Accordion.tsx:4]: Comment this

File: src/components/layouts/molecules/Accordion.tsx [hunk 10]
```diff
@@ -0,0 +1,24 @@
...
+  const label = props.molecule.staticValues.label || undefined;
+  return (
+    <Accordion.Toggle as={Button} variant="link" eventKey="0">
+      {label}
+    </Accordion.Toggle>
+  )
+}
+
+export function AccordionMolecule(props: any) {
```
Comment (hey-waffles) [inline src/components/layouts/molecules/Accordion.tsx:13]: Comment

File: src/components/layouts/molecules/index.tsx [hunk 11]
```diff
@@ -0,0 +1,44 @@
...
+import { MoleculeType } from "../../../types/enums";
+import { Atom } from "../../../types/layouts";
+import { Molecule } from "../../../types/layouts/molecule";
+import { renderDisplayClasses } from "../../../utilities/layouts/atoms";
+import { DynamicAtom } from "../atoms";
+import { AccordionMolecule } from "./Accordion";
+
+
+
+interface MoleculeProps {
```
Comment (hey-waffles) [inline src/components/layouts/molecules/index.tsx:11]: Comment

File: src/pages/search/[gameSystemAlias]/index.tsx [hunk 12]
```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import Page from "../../../components/design/Page";
+
+export default function GameSystemSearch(props: any) {
```
Comment (hey-waffles) [inline src/pages/search/[gameSystemAlias]/index.tsx:4]: Delete this page??

**Review by lucyawrey — APPROVED** [review]
Looks good

**Pull request merged**